lxc/config/remote.go

```diff
@@ -115,7 +115,7 @@ func (c *Config) GetInstanceServer(name string) (lxd.InstanceServer, error) {
 }
 
 // GetImageServer returns a ImageServer struct for the remote
-func (c *Config) GetImageServer(name string) (lxd.ImageServer, error) {
+func (c *Config) GetImageServer(name string, allowProjectOverride bool) (lxd.ImageServer, error) {
```

Comment:
We can't do this, this is public API and would cause some major breakages to other software using the LXD client Go code.